src/aircloak/AircloakApi/ResponseTypes.cs

```diff
@@ -170,6 +170,17 @@ public struct QueryDataSource
     public struct QueryResult<RowType>
     {
         public QueryResultInner<RowType> Query { get; set; }
+
+        /// <summary>
+        /// Extract the rows from the innards of the result type. 
+        /// </summary>
+        public IEnumerable<RowType> ResultRows()
```

Comment:
I think that you can make this a property, and this will be more like idiomatic C#.

Reply:
For this I wasn't sure if using a property would interfere with the automatic JSON deserializing (ie. the serializer might look for a 'result_rows' property in the json response. I'll test it.

Reply:
I think that you could use the [JsonIgnore] attribute on the property to avoid the issue.

Reply:
I changed it to this: 
```
        [JsonIgnore]
        public IEnumerable<TRow> ResultRows
        {
            get
            {
                foreach (var row_with_count in Query.Rows)
                {
                    yield return row_with_count.Row;
                }
            }
        }
```

Seems to work just as well.